packages/webapp/src/elections/interfaces.ts

```diff
@@ -1,45 +1,75 @@
+import { string } from "zod";
```

Comment:
Did you really mean to use string from zod? If yes, what is zod behavior in the following interfaces?

Reply:
I definitely did *not*. this must have been an autocomplete import I didn't see happen. Great catch @sparkplug0025 !